### ISSUE TITLE
startup errors after update to 3.0.0 with redis adapter

### ISSUE THREAD

**Issue by rabalyn** [feathersjs-ecosystem/feathers-sync#178]
### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))

- [ ] Tell us what broke. The more detailed the better.
- [ ] If you can, please create a simple example that reproduces the issue and link to a gist, jsbin, repo, etc.

Hi there,

after the update from `feathers-sync@2.4.0` to `feathers-sync@3.0.0` the app start is producing some errors, but the module connects to redis and is emitting events.

So, maybe just an internal initialization issue?

Tested connection strings:

```
// I tested following connection strings
app.get('redis').uri === "redis://default:superSecret@localhost:6379"
app.get('redis').uri === "redis://default:superSecret@localhost:6379/1"
// no user
app.get('redis').uri === "redis://:superSecret@localhost:6379"
app.get('redis').uri === "redis://:superSecret@localhost:6379/1"
```

Initialization:

```js
app.configure(
  sync({
    uri: app.get('redis').uri
  })
)
```

### Expected behavior

- app should start with no error
- debug message should output the actual connection string instead of `undefined`

### Actual behavior

```text
[nodemon] starting `node src/`
  feathers-sync:redis Setting up Redis client for undefined +0ms
  feathers-sync:redis redisClient: undefined +3ms
info: Feathers application started on http://localhost:3030
Redis connected
error: Unhandled Rejection at: Promise
AuthError: Cannot send commands in PubSub mode
    at ~~~/server/node_modules/@node-redis/client/dist/lib/client/index.js:323:23
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0)
    at async RedisSocket.socketInitiator (~~~/server/node_modules/@node-redis/client/dist/lib/client/index.js:332:13)
    at async RedisSocket._RedisSocket_connect (~~~/server/node_modules/@node-redis/client/dist/lib/client/socket.js:130:13)
    at async Commander.connect (~~~/server/node_modules/@node-redis/client/dist/lib/client/index.js:162:9)
  feathers-sync:redis Publishing key feathers-sync to Redis +1s
  feathers-sync:redis Publishing key feathers-sync to Redis +49s
  feathers-sync:redis Publishing key feathers-sync to Redis +10s
```

### System configuration

Redis Version: redis:6.2.5-alpine

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):
```json
{
  "dependencies": {
    "@feathersjs/authentication": "^4.5.12",
    "@feathersjs/authentication-local": "^4.5.12",
    "@feathersjs/configuration": "^4.5.12",
    "@feathersjs/errors": "^4.5.12",
    "@feathersjs/express": "^4.5.12",
    "@feathersjs/feathers": "^4.5.12",
    "@feathersjs/socketio": "^4.5.13",
    "@feathersjs/transport-commons": "^4.5.12",
    [...]
    "feathers-sync": "^3.0.0",
    [...]
  }
}
```

**NodeJS version**:

node v16.13.1

**Operating System**:

Ubuntu 20.04

**Browser Version**:

**React Native Version**:

**Module Loader**:

**Comment by daffl** [comment]
The `undefined` in `Setting up Redis client for undefined` part is what I think points at what I'm pretty sure is a bug. I bet the tests passed in CI because it was then using the default connection string.

**Comment by rabalyn** [comment]
Well, the `undefined` debug message is a typo in the string literal in [redis.js#L14](https://github.com/feathersjs-ecosystem/feathers-sync/blob/0359e5c516395580e5808e3edaf7c65904860593/lib/adapters/redis.js#L14)

Which should be `${options.url}` instead of `${options.uri}`

But I can't see what could lead to the `error` and `AuthError`

**Comment by daffl** [comment]
Closed via #180
